addons/trenches/stringtable.xml

```diff
@@ -239,5 +239,41 @@
             <Chinese>移除掩體中</Chinese>
             <Turkish>Siper Kaldırılıyor</Turkish>
         </Key>
+        <Key ID="STR_ACE_Trenches_Category">
+            <English>ACE Trenches</English>
+            <French>ACE Tranchées</French>
+        </Key>
+        <Key ID="STR_ACE_Trenches_SmallEnvelopeDigDuration_DisplayName">
+            <English>Small trench dig duration</English>
+            <French>Durée d'excavation d'une petite tranchée</French>
+        </Key>
+        <Key ID="STR_ACE_Trenches_SmallEnvelopeDigDuration_Description">
+            <English>Small trench dig duration in seconds.</English>
```

Comment:
Maybe: "Time, in seconds, required to dig a small trench."

Reply:
Updated